cmd/esc/cli/env.go

```diff
@@ -72,20 +72,27 @@ type environmentRef struct {
 	version     string
 
 	hasAmbiguousPath bool
+	isRelative       bool
```

Comment:
Can you add a docstring here? I'm having a hard time following when something would be relative